src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java

```diff
@@ -29,31 +28,29 @@
   @BeforeMethod(alwaysRun = true)
   public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
-
-    //@TODO XW-314 - Login is necessary to bypass cache
-    new LoginPageObject(driver).get().logUserIn(Configuration.getCredentials().userNameStaff2,
-                                                Configuration.getCredentials().passwordStaff2);
   }
 
+  //@TODO XW-314 - Login is necessary to bypass cache.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)